tests/scripts/thread-cert/Cert_8_1_01_Commissioning.py

```diff
@@ -46,7 +51,7 @@ def setUp(self):
 
         self.nodes[COMMISSIONER].set_panid(0xface)
         self.nodes[COMMISSIONER].set_mode('rsdn')
-        self.nodes[COMMISSIONER].set_masterkey('deadbeefdeadbeefdeadbeefdeadbeef')
+        self.nodes[COMMISSIONER].set_masterkey('00112233445566778899aabbccddeeff')
```

Comment:
I was not setting master keys the same, but set COMMISSIONER's master key to the default `00112233445566778899aabbccddeeff ` so that it would be the master key of the network an d we could decrypt captured traffic with wireshark.

And, as the test plan does not require the joiner be set to a different master key, I didn't change it.